LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2020 DNS Made Easy, a subsidiary of Tiggee LLC.
+Copyright (c) 2021 Sandwave.io B.V.

```

Comment:
I think you need to preserve the old license.

> The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

Reply:
Very good point, I have fixed this.